src/stream/src/executor/debug/epoch_check.rs

```diff
@@ -46,8 +46,15 @@ pub async fn epoch_check(info: Arc<ExecutorInfo>, input: impl MessageStream) {
                 );
             }
 
-            if let Some(last_epoch) = last_epoch {
-                assert!(b.epoch.prev == last_epoch, "missing barrier: last barrier's epoch = {}, while current barrier prev={} curr={}", last_epoch, b.epoch.prev, b.epoch.curr);
+            if let Some(last_epoch) = last_epoch && !b.is_with_stop_mutation() {
```

Comment:
Are all actors to be stopped in fail-over? If so, it seems we can use `is_stop_or_update_drop_actor(info.actor_id)` here.

Reply:
Yes, all actors is gonna be stopped in failover. And the reason why I didn't use `is_stop_or_update_drop_actor` is because I still want to keep the epoch check in online scaling.